recipes/go-activation/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% if version is undefined %}
+{% set version = "1.16.7" %}
+{% endif %}
+
+{% if cross_target_platform is undefined %}
+{% set cross_target_platform = "foo" %}
+{% endif %}
+
+package:
+  name: go-{{ go_variant_str }}_{{ cross_target_platform }}
+  version: {{ version }}
+
+build:
+  number: 1
+  skip: True  # [linux and ((cross_target_platform == "osx-arm64") or (cross_target_platform == "osx-64")) and cgo]
```

Comment:
Why this skip?

Reply:
I thought that we cannot compile C code from Linux to OSX but actually, we can, so we can remove this skip.
```suggestion
```